MennaMohie/assignment1/Q1.cpp

```diff
@@ -0,0 +1,134 @@
+#include<bits/stdc++.h>
+#include<stdlib.h>
+#define endl '\n'
```

Comment:
Why ?
The standard library already defines a endl

Reply:
It's because <<'\n' is faster than <<endl but endl is easier to type so I defined it as '\n' because it used to make a difference when I solved problems on CodeForces, but I will remove it. :D